Packs/Akamai_SIEM/ReleaseNotes/1_1_7.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Akamai WAF SIEM
+
+- Fixed an issue where **fetch-events** command failed with docker time out error.
```

Comment:
Fixed an issue where the **fetch-events** command failed with a docker time out error.